enscons/__init__.py

```diff
@@ -187,7 +183,7 @@ def egg_info_builder(target, source, env):
     """
     Minimum egg_info. To be used only by pip to get dependencies.
     """
-    metadata = env["PACKAGE_METADATA"]
+    env["PACKAGE_METADATA"]
```

Comment:
This deserves a comment since it doesn't seem like it does anything (and in fact, I'm not really sure what it does).